### ISSUE TITLE
How to give a lifetime to Bodies?

### ISSUE THREAD

**Issue by deepika08** [liabru/matter-js#1044]
I want to create a shooting game where my bullets should get removed after a certain time.
I have removed gravity as it affects the trajectory of bullets.
I tried setTimeOut but this gets applied on all bullets in future too.

**Comment by Justiniscoding** [comment]
Whenever you create a new bullet you can use `setTimeout(/* time in milliseconds until you want a bullet to be deleted */,() => /* code to remove bullet */);`

**Comment by liabru** [comment]
Yeah you should be able to use `setTimeout` and `Composite.remove` to do this?
